src/chrome/chromeDebugAdapter.ts

```diff
@@ -839,10 +839,14 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
 
             const resolvePendingBPs = async (source: string) => {
                 source = source && utils.canonicalizeUrl(source);
-                const pendingBP = this._pendingBreakpointsByUrl.get(utils.fixDriveLetter(source)) || this._pendingBreakpointsByUrl.get(utils.fixDriveLetter(source, true));
+                logger.log(`OnScriptParsed.resolvePendingBPs: pendingBreakpointsByUrl = ${JSON.stringify([...this._pendingBreakpointsByUrl])} source = ${source}`);
```

Comment:
I don't think you need to copy `this._pendingBreakpointsByUrl` (just JSON.stringify(this. ...)

Reply:
Oh it's a map, I see. Do you need to log the whole thing, and the individual bp below?

Reply:
It was useful while I was debugging the test. I'm not sure what logging we'll need for the next issue.

Reply:
This message will be printed for every loaded script whether or not there are pending BPs, right? Can you cut it down to just the necessary stuff to avoid too much log spam?

Reply:
You are right, so much spam will be annoying. What do you think of this filter?